src/coreclr/pal/src/misc/perfjitdump.cpp

```diff
@@ -2,7 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // ===========================================================================
 
-#if defined(__linux__)
+#if defined(__linux__) || defined(__APPLE__)
```

Comment:
Would it make sense to enable this unconditionally? (This file is for non-Windows, so enabling it unconditionally here would mean enabling it unconditionally for all non-Windows OSes.) 

`__APPLE__` means all Apple OSes, so this is technically enabling it on more than just macOS - if we start compiling CoreCLR for more than just macOS.

Reply:
> start compiling CoreCLR for more than just macOS

#98127